server/sonar-web/src/main/js/apps/coding-rules/routes.js

```diff
@@ -21,9 +21,9 @@ const routes = [
   {
     indexRoute: {
       getComponent(_, callback) {
-        require.ensure([], require => {
-          callback(null, require('./components/CodingRulesAppContainer').default);
-        });
+        import('./components/CodingRulesAppContainer').then(i =>
+          callback(null, { component: i.default })
```

Comment:
Are you sure we must return an object with the `component` field here ? Seems like we were returning the component directly with the `require.ensure`